Snakefile

```diff
@@ -1090,6 +1087,7 @@ rule decorate_expression_baseline:
         set -e # snakemake on the cluster doesn't stop on error when --keep-going is set
         exec &> "{log}"
         source {workflow.basedir}/bin/reprocessing_routines.sh
+        source {workflow.basedir}/atlas-bash-util/generic_routines.sh
```

Comment:
if you are only adding a source here, how were these variables provisioned before? The ones that you are getting from bash-util/generic_routines?

Reply:
The variables are provisioned as before. Specifically (see below) for 
```
geneNameFile=$( get_geneNameFile_given_organism {params.organism}  )
```
`find_properties_file` now is sourced from atlas-bash-util/generic_routines.sh

Reply:
I did, however, found another case where a variable needed to be exported [c2e555f](https://github.com/ebi-gene-expression-group/bulk-recalculations/pull/40/commits/c2e555f44c70b977d64ab14ee591da123e55be6f)